### DISCUSSION TITLE
Cron job stop without any exception.

### DISCUSSION THREAD

**Discussion by bsdeepak009** [quartznet/quartznet#1952] [Q&A]
![image](https://user-images.githubusercontent.com/97017533/225326738-96865c86-8170-4bce-b3ec-f5edf981c5cd.png)
```
try
            {
                var scheduler = await StdSchedulerFactory.GetDefaultScheduler();
                await scheduler.Start();

                IJobDetail job = JobBuilder.Create<WatchListsCommand>().WithIdentity("myJob", "group1")
                    .Build();
                ITrigger trigger = null;
                string defaultTime = "0 10 21 1/1 * ? *";
                
                //string defaultTime = "0 0/1 * 1/1 * ? *";
                trigger = TriggerBuilder.Create()
                                           .WithCronSchedule(defaultTime)
                                           .Build();

                await scheduler.ScheduleJob(job, trigger);

            } catch(Exception ex)
            {
                logger.Error(ex);
            }
```
            
            this cron job running at 8:00 PM and some time it's stop without any exception please help ?
            
            
![image](https://user-images.githubusercontent.com/97017533/225327519-11e85698-39ce-4f62-b062-3313d326963b.png)

**Comment by lahma** [comment]
You should at least a keep a reference to scheduler instead of having it just as method scope variable. Also consider enabling logging to see what happens under the hood.

**Comment by bsdeepak009** [comment]
Please give any example?
